packages/graphql/tests/schema/directives/relationship-properties.test.ts

```diff
@@ -50,6 +50,10 @@ describe("Relationship-properties", () => {
               mutation: Mutation
             }
 
+            \\"\\"\\"
+            Type describing relationship properties for the Actor.movies field.
+            Type describing relationship properties for the Movie.actors field.
```

Comment:
Well they are 2 different fields and they don't need be.. I don't think it would be fair to have only one of the 2 sentences. 
Also, the text is out of my imagination and I'm sure could be better 😄

Reply:
:thinking: 
I was thinking that maybe there is a way of describing the type for both fields a bit better, as it may be confusing having those 2 descriptions, but I'm not sure, it is not a big problem

Reply:
I would if we could have something like:

```
"""
The edge properties for the following fields:

* Actor.movies
* Movie.actors
"""
```

I imagine harder to generate though!!

Reply:
Should not be that hard, I'm implementing this